grammar.js

```diff
@@ -1,4 +1,4 @@
-const C = require("../tree-sitter-c/grammar")
+const C = require("tree-sitter-c/grammar")
```

Comment:
@calixteman When developing `tree-sitter-cpp`, I usually use `npm link` to ensure that my local copy of `tree-sitter-c` is used. That way you don't need to change this path temporarily in order to use local changes.